### PR TITLE
Add user profile edit page

### PR DESCRIPTION
**Pull request by richtier**
[this task](https://uktrade.atlassian.net/browse/ED-227)

image

### CODE REVIEW

File: user/forms.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+from django import forms
+
+
+class UserBasicInfoForm(forms.Form):
+    name = forms.CharField()
+    email = forms.URLField()
```
Comment (konradko) [inline user/forms.py]: URL?

File: user/forms.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+
+def serialize_user_profile_forms(cleaned_data):
+    """
```
Comment (konradko) [inline user/forms.py]: Google's docstring style is preferred for the project: http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

Fine to leave it as it is for now, good it has a docstring :-)

File: user/templates/user-profile-update-error.html [hunk 3]
```diff
@@ -0,0 +1,4 @@
+{% extends 'registration-base.html' %}
+
+{% block hero_title %}There was a problem updating your profile.{% endblock %}
```
Comment (konradko) [inline user/templates/user-profile-update-error.html]: hero?

File: user/forms.py (hunk 1 above)
Comment (konradko) [inline user/forms.py]: this reminds me - this should be 'business_email/corporate_email', as user will have another one from SSO too (which is less restrictive)

File: user/templates/user-profile-update-error.html (hunk 3 above)
Comment (konradko) [inline user/templates/user-profile-update-error.html]: ah alright

File: user/forms.py (hunk 1 above)
Comment (konradko) [inline user/forms.py]: Looks like this still needs changing

File: user/forms.py (hunk 2 above)
Comment (richtier) [inline user/forms.py]: There are a few docstrings needing changes. will do as one follow up PR

**Pull request merged**